src/tox_wheel/plugin.py

```diff
@@ -72,10 +72,7 @@ def tox_package(session, venv):
 
 
 def wheel_build_package(config, session, venv):
-    if config.isolated_build:
-        reporter.warning("Disabling isolated_build, not supported with wheels.")
-    pep517 = venv.envconfig.wheel_pep517
-    if pep517:
+    if config.isolated_build or venv.envconfig.wheel_pep517:
```

Comment:
I typoed this and didn't notice:

https://github.com/twisted/ldaptor/pull/180/files#diff-522adf759addbd3b193c74ca85243f7dR31

Reply:
> Why did I put that warning I wonder ...



> Why did I put that warning I wonder ...

@ionelmc I'm not sure, it *seems* sensible to me to process the flag for its stated purpose rather than claim to disable it while not disabling it

Reply:
I presume the warning predates wheel_pep517?